stdlib/src/locale.rs

```diff
@@ -0,0 +1,101 @@
+#[cfg(unix)]
+pub(crate) use _locale::make_module;
+
+#[cfg(unix)]
+#[pymodule]
+mod _locale {
+    use rustpython_vm::{
+        builtins::{PyDictRef, PyIntRef, PyListRef},
+        PyObjectRef, PyResult, VirtualMachine,
+    };
+
+    #[pyattr]
+    use libc::{
+        ABDAY_1, ABDAY_2, ABDAY_3, ABDAY_4, ABDAY_5, ABDAY_6, ABDAY_7, ABMON_1, ABMON_10, ABMON_11,
+        ABMON_12, ABMON_2, ABMON_3, ABMON_4, ABMON_5, ABMON_6, ABMON_7, ABMON_8, ABMON_9,
+        ALT_DIGITS, AM_STR, CODESET, CRNCYSTR, DAY_1, DAY_2, DAY_3, DAY_4, DAY_5, DAY_6, DAY_7,
+        D_FMT, D_T_FMT, ERA, ERA_D_FMT, ERA_D_T_FMT, ERA_T_FMT, LC_ALL, LC_COLLATE, LC_CTYPE,
+        LC_MESSAGES, LC_MONETARY, LC_NUMERIC, LC_TIME, MON_1, MON_10, MON_11, MON_12, MON_2, MON_3,
+        MON_4, MON_5, MON_6, MON_7, MON_8, MON_9, NOEXPR, PM_STR, RADIXCHAR, THOUSEP, T_FMT,
+        T_FMT_AMPM, YESEXPR,
+    };
```

Comment:
Not all of these constants are available on every platform. When I encountered this sort of problem in the past, I looked up each item individually in the `libc` repository to see which platforms support that item. (See #3840, for example.)

I realized back then (and I still maintain) that this process would prove to be tedious and error-prone. Perhaps I should look into using the `autocfg` crate instead, but that's ultimately beyond the scope of this PR.

Reply:
I tried with Window locale.h this afternoon and found out that Window API uses different naming convention (prefix with W, if I remember). Hence, if we would like to fix it, I'm afraid that we need to implement our own locale interaction. I will do it in another PR once this module takes shape.